src/zennit/rules.py

```diff
@@ -57,12 +57,33 @@ class Gamma(BasicHook):
     '''
     def __init__(self, gamma=0.25, zero_params=None):
         super().__init__(
-            input_modifiers=[lambda input: input],
-            param_modifiers=[lambda param, _: param + gamma * param.clamp(min=0)],
-            output_modifiers=[lambda output: output],
-            gradient_mapper=(lambda out_grad, outputs: out_grad / stabilize(outputs[0])),
-            reducer=(lambda inputs, gradients: inputs[0] * gradients[0]),
-            zero_params=zero_params,
```

Comment:
I don't think it's intentional: here you completely remove the `zero_params` parameter. So the `zero_params` argument is never used or passed to the base class. Just keep this in mind :smile:.

Reply:
Sorry, the changes were just not pushed yet.
The parameter in the `__init__` of remains unchanged, only of `BasicHook`.

Reply:
Thanks for clarifying and sorry for the circumstances. Now of course it makes more sense. I had already suspected that this will come with a separate pull request.